src/Microsoft.ML.TimeSeries/STL/SeasonalInfo.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.TimeSeries
+{
+    public class SeasonalInfo : SingleSeriesInfo
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SeasonalInfo"/> class.
+        /// the corresponding properties of seasonal component
+        /// </summary>
+        /// <param name="x">x</param>
+        /// <param name="y">y</param>
+        /// <param name="seasonal">the seasonal component</param>
+        /// <param name="period">period of seasonal component</param>
+        /// <param name="amplitude">the average amplitude of the seasonal component</param>
+        /// <param name="confidence">the statistical confidence</param>
+        public SeasonalInfo(
+            IReadOnlyList<double> x,
+            IReadOnlyList<double> y,
+            IReadOnlyList<double> seasonal,
```

Comment:
It seems this constructor is not called anywhere. Please see my comment here:
https://github.com/dotnet/machinelearning/pull/5202/files?file-filters%5B%5D=.cs&file-filters%5B%5D=.csv#r440494174

Reply:
Removed this file.

---
In reply to: [440513863](https://github.com/dotnet/machinelearning/pull/5202#discussion_r440513863) [](ancestors = 440513863)